src/gui_common/GUICommon.cs

```diff
@@ -89,6 +89,23 @@ public static void SmoothlyUpdateBar(TextureProgress bar, float target, float de
         bar.Value = MathUtils.Lerp((float)bar.Value, target, weight, 1.0f / (float)bar.MaxValue);
     }
 
+    /// <summary>
+    ///   Loads a Texture from predefined GUI asset texture folder path.
+    /// </summary>
+    public static Texture LoadGuiTexture(string file)
+    {
+        var assumedPath = PathUtils.Join(Constants.ASSETS_GUI_BEVEL_FOLDER, file);
+
+        if (ResourceLoader.Exists(assumedPath, "Texture"))
+            return GD.Load<Texture>(assumedPath);
+
+        // Fail-safe if file itself is the absolute path
+        if (ResourceLoader.Exists(file, "Texture"))
+            return GD.Load<Texture>(file);
+
+        return null;
```

Comment:
I think attempting to load anyway would be better than returning null, that way Godot can log the missing texture file.